backend/expense-calculator/src/main/java/org/egov/digit/expense/calculator/web/controllers/WorksCalculatorApiController.java

```diff
@@ -73,4 +71,14 @@ public ResponseEntity<CalculationResponse> worksCalculatorV1EstimatePost(
 		return new ResponseEntity<CalculationResponse>(calculationResponse, HttpStatus.OK);
 	}
 
+
+	@RequestMapping(value = "/v1/_search", method = RequestMethod.POST)

```

Comment:
you can call it calculator search when the method is just returning bill-ids, call it billId search. instead of billids you can directly call expense service and fetch all bills and respond(check with jaga on what makes things easier for everyone).